### ISSUE TITLE
Isort 5.4.0 with combine-as drops comments on some imports

### ISSUE THREAD

**Issue by ziima** [timothycrosley/isort#1381]
When applying isort on a following snippet
```python
from smtplib import SMTPConnectError, SMTPNotSupportedError  # important comment
```
isort tries to drop the comment.

Command:
```
isort --line-length=120 --check-only --diff --combine-as /tmp/isort_comment.py
```

Affected version is 5.4.0, comment is untouched by isort 5.3.2. As far as I know, only this particular import is affected, I wasn't able to create another case. But the bug was reproduced on several machines.

**Comment by tpazderka** [comment]
This can be triggered by
```python
from appsettings import AppSettings, ObjectSetting, StringSetting  # type: ignore
```
as well.

**Comment by timothycrosley** [comment]
Thank you for reporting, I'm sorry this regression made it's way through! I've added an additional regression test that tests both of these cases to the isort test regression test suite and made pushed a hot fix release (5.4.1) to address this regression.

Thanks!

~Timothy
